### ISSUE TITLE
More recent references

### ISSUE THREAD

**Issue by nnadeau** [ross-rotordynamics/ross#479]
Many of the references cited in [`paper/paper.md`](paper/paper.md) are old. Unless the reference is a seminal work, references should be <5 years old. Are there other open source software frameworks for rotordynamics?

From https://academia.stackexchange.com/a/24050 :

- How can the reviewer judge the contribution if recent work isn't cited? 
- Has what's being proposed already been done? In a better way?
- Not acknowledging recent works can be seen as lazy and potentially misleading.

cc https://github.com/openjournals/joss-reviews/issues/2120

**Comment by raphaeltimbo** [comment]
The rotordynamic theory that we are using to implement ROSS is standard and has changed little in recent years, that is why the references we have used are not so recent. 

We only know of one open source code for rotordynamics which is cited in the paper (Friswell, Michael I., et al. Dynamics of rotating machines. Cambridge university press, 2010). But this code needs MATLAB to run and it is developed more as an educational tool. As described in their manual: 

_“This software is a set of scripts written in MATLAB to accompany the above book. The primary purpose of the software is to illustrate features of rotating machines described in the book, in particular the lateral motion based on shaft-line models._” 

The purpose of ROSS is to have a software that follows the open source ideas. To achieve that we have developed the code with a free and open source language (Python), using collaboration tools as GitHub. 

ROSS can be used with engineering purposes or with research purposes, but in the ROSS development we have not applied any novel mathematical/modeling theory that is different from that used in others softwares that we have referenced at the paper.
